manifests/rules/out/chrony.pp

```diff
@@ -1,7 +1,28 @@
-# manage out chrony
-class nftables::rules::out::chrony {
-  nftables::rule {
-    'default_out-chrony':
-      content => 'udp dport 123 accept',
+# @summary manage out chrony
+# @param servers single IP-Address or array of IP-addresses from NTP servers
+class nftables::rules::out::chrony (
+  Variant[Stdlib::IP::Address,Array[Stdlib::IP::Address]] $servers = [],
+) {
+  $_servers = [$servers].flatten
+  if empty($_servers) {
+    nftables::rule {
+      'default_out-chrony':
+        content => 'udp dport 123 accept',
+    }
+  } else {
+    $_servers.each |$index,$ps| {
+      nftables::rule {
```

Comment:
Rather than creating a rule per server you could consider creating an ip set instead?

Reply:
Not insisting since you are not going to have lots of time servers typically.